design-library/src/components/BccInfo/BccInfo.stories.ts

```diff
@@ -0,0 +1,43 @@
+import BccInfo from "./BccInfo.vue";
+import BccButton from "../BccButton/BccButton.vue";
+import { ClockLoader10Icon } from "@bcc-code/icons-vue";
+import type { Meta, StoryFn } from "@storybook/vue3";
+
+export default {
+  title: "Components/BccInfo",
+  component: BccInfo,
+  argTypes: {},
+} as Meta<typeof BccInfo>;
+
+const Template: StoryFn<typeof BccInfo> = (args) => ({
+  components: { BccInfo, BccButton },
+  setup() {
+    return { args };
+  },
+  template: `
+    <BccInfo v-bind="args">
+        <template #infoRight>
+        <BccButton size="sm">Button text</BccButton>
+      </template>
+    </BccInfo>
+  `,
+});
+
+export const Example = Template.bind({});
```

Comment:
This story gives an error when I click "View code":
```
SyntaxError: Unexpected closing tag "BccInfo"
```

Reply:
Weird storybook issue.